ts/tat-trend.ts

```diff
@@ -371,6 +433,174 @@ function updatePlotWithLegend(
   }
 }
 
+function updateMetricsTable(
+  jsonData: any[],
+  selectedGrouping: string,
+  selectedGates: string[],
+  selectedDataType: string
+) {
+  // build the table data and get the time ranges from the metrics
+  const { tableData, timeRanges } = buildTableFromMetrics(
+    jsonData,
+    selectedGrouping,
+    selectedGates,
+    selectedDataType
+  );
+  const sortedTimeRanges = sortTimeRanges(timeRanges);
+  const dynamicColumns = generateDynamicColumns(sortedTimeRanges);
+  const parentHeaders = [
+    { title: "", colspan: 1 },
+    { title: "", colspan: 1 },
+    ...sortedTimeRanges.map((timeRange) => ({
+      title: timeRange,
+      colspan: 3,
+    })),
+  ];
+  const caseTatTableDefinition: TableDefinition<AssayMetrics, void> = {
+    generateColumns: () => dynamicColumns,
+    parentHeaders, // pass parent headers for multi-level header support
+    disablePageControls: true,
```

Comment:
You've set sorting options for a couple columns, but this removes the sorting function